microhapdb/tests/test_nomenclature.py

```diff
@@ -64,7 +64,7 @@ def test_hyphen():
 
 
 def test_lab():
-    id1 = Identifier("mh16DSSAGS-123")
+    id1 = Identifier("mh16DSSAGSRNM-123")
```

Comment:
Updated this test to trigger the expected validation failure due to a long lab label.